app/scripts.babel/friends.js

```diff
@@ -204,4 +209,12 @@ init({ friends: true, cachedRanks: true }, ({ friends, cachedRanks }) => {
       replaceRanklist();
     }
   });
+}
+
+observer.forEach(SELECTOR_TO_HIDE, elem => elem.style.display = 'none');
```

Comment:
This hides the title of the page. I propose hiding only the `.ranklist` and doing the `pageBody` stuff here (see above).